ci/infra/testrunner/testrunner

```diff
@@ -22,6 +22,5 @@ setup_python_env() {
 }
 
 setup_python_env
-# use unbuffered output
-stdbuf -i0 -o0 -e0 python -u ${SDIR}/testrunner.py "$@"
```

Comment:
Ugly as it is, I think we might like to leave the `stdbuf` commands. There's some desire (SUSE/avant-garde#670) to have tests running within Jenkins or similar generate synchronous output.  Imagine something like wget which prints progress output before a newline happens, for example.  That, of course, will also require a tweak to how the `read_fd` threads read from the pipe, but we can deal with that later.  This gets us 90% there.

Reply:
I would like to only keep things we really need, otherwise we can just keep certain changes because they were there before without a real use. Honestly, if we need to have a complete unbuffered output let's do that in the future, in my opinion.